script/checkLocale.mjs

```diff
@@ -0,0 +1,105 @@
+import fs from 'fs/promises'
+import path from 'path'
+
+const CONSTANTS = {
+  BASE_LOCALE: 'en',
+  LOCALE_DIR: 'locales',
+  LOCALES: ['en', 'fr', 'de', 'es'], // "ALL" or locales needed eg. ["fr","de"]
```

Comment:
> can we make this depending on the list of files in `locales` folder?

you mean get all locales from `locales` folder?  hmm there we can set the `LOCALES:  'ALL'` to do so, but i leave this configuration here because i noticed that we are only use these 4 locales now.